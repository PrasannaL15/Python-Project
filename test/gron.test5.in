{
  "name": null,
  "age": null,
  "email": null
}
