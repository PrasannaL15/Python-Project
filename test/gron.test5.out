json = {}
json.age = null
json.email = null
json.name = null

